canal/handler.go

```diff
@@ -11,7 +11,7 @@ type EventHandler interface {
 	OnRow(e *RowsEvent) error
 	OnXID(nextPos mysql.Position) error
 	OnGTID(gtid mysql.GTIDSet) error
-	SaveMasterInfo(pos mysql.Position, force bool) error // Use your own way to save master info
+	OnPosSynced(pos mysql.Position, force bool) error // Use your own way to save master info
```

Comment:
remove the latter comment. 

Btw, we should add a comment for the force argument.